compute_tools/src/spec.rs

```diff
@@ -511,17 +511,26 @@ pub fn handle_grants(spec: &ComputeSpec, connstr: &str, client: &mut Client) ->
         .map(|r| r.name.pg_quote())
         .collect::<Vec<_>>();
 
-    for db in &spec.cluster.databases {
-        let dbname = &db.name;
+    let is_webaccess = spec
+        .cluster
+        .roles
+        .iter()
+        .map(|r| r.name == "web_access")
```

Comment:
why do you need `map` if you can put this comparison directly into `any`?

```rust
.any(|r| r.name == “web_access”)
```

Reply:
Good point, I'm still new to this rust thingy